global/scripts/gitleaks/.gitleaks.toml

```diff
@@ -556,6 +556,16 @@ id = "Social Security Number"
 description = "Social Security Number"
 regex = '''\d{3}-\d{2}-\d{4}'''
 
+[[rules]]
+id = "Confidential UUID"
+description = "Hardcoded Application ID (UUID)"
+regex = '''[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}'''
+
+[[rules]]
+id = "Application Secret"
+description = "Hardcoded Application Secret with getEnv check"
+regex = '''(?i)getEnv\s*\(\s*["']APPLICATION_SECRET["']\s*,\s*["'][A-Za-z0-9._~\-]{32,}["']\s*\)'''
```

Comment:
Here, seems to be a GoLang function. If it's specific for GoLang, I recommend you to create more patterns of regex to capture Java, Python as well.

Reply:
Added the rules for java and python as well.